public/app/plugins/datasource/elasticsearch/datasource.test.ts

```diff
@@ -274,6 +274,85 @@ describe('ElasticDatasource', function(this: any) {
     });
   });
 
+  describe('When getting an error on response', () => {
+    const query = {
+      range: {
+        from: toUtc([2020, 1, 1, 10]),
+        to: toUtc([2020, 2, 1, 10]),
+      },
+      targets: [
+        {
+          alias: '$varAlias',
+          bucketAggs: [{ type: 'date_histogram', field: '@timestamp', id: '1' }],
+          metrics: [{ type: 'count', id: '1' }],
+          query: 'escape\\:test',
+        },
+      ],
+    };
+
+    createDatasource({
+      url: 'http://es.com',
```

Comment:
As we just spoke about, this should probably be changed, since `es.com` doesn't have anything to do with elasticsearch.
I also wonder whether it should look like a real URL at all. Because when looking at these tests, it might be good to convey to whoever is reading them that this `url` doesn't actually mean anything, since we mock the responses anyway.

Reply:
Replaced with `http://elasticsearch.local`.